common/src/main/resources/css/charts.scss

```diff
@@ -1,5 +1,4 @@
-@import 'https://code.highcharts.com/css/highcharts.css';
-/* TODO Can we import from the facade package? */
+@use "~highcharts/css/highcharts.scss";
```

Comment:
Is use, import in sass?

Reply:
@use is a "new @import" (@import is also supported but deprecated). @use imports things only one even if used multiple times, imported things are namespaced, plus other features for better modularization. More here: https://css-tricks.com/introducing-sass-modules/#import-files-with-use